<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnDodajPacjenta">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj Pacjenta</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>821</width>
      <height>351</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Imie</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nazwisko</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Płeć</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wiek</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data Urodzenia</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pesel</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Użytkownik</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Specjalizacja</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Kartoteka Urzytkowników</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz użytkowników do .txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUsun">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń użytkownika</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDodajLekarza">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj Lekarza</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDodajWizyte">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj do wizyty Pacjenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImie">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNazwisko">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>570</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dane pacjenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="imieLekarza">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNazwiskLekarza">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dane lekarza </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj do wizyty Lekarza</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWizyty">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>731</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Imie Pacjenta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nazwisko Pacjenta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data Wizyty</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Opis Dolegliwości</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Imie Lekarza</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nazwisko Lekarza</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>WIZYTA</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDodajWizyteInfo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>620</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj Wizytę</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>1061</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>10</y>
      <width>20</width>
      <height>901</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>900</y>
      <width>1061</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>20</width>
      <height>901</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>1061</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWizyty">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>710</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz wizyty do pliku .txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZaladuj">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Załaduj uzytkowników</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWczytajWizyty">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>780</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj wizyty</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
